File: YuGiOhCard/CardAdjustViewController.m
#import "CardAdjustViewController.h"
#import "CardViewController.h"
#import "CardItem.h"

@interface CardAdjustViewController ()

@property (nonatomic) CardItem * card;

@end

@implementation CardAdjustViewController

- (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
{
    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
    if (self) {
        // Custom initialization
    }
    return self;
}

- (void)viewDidLoad
{
    [super viewDidLoad];
    self.card = ((CardViewController *)self.tabBarController).card;
    NSString * str = [NSString stringWithFormat:@"%@\n\n\n\n", self.card.adjust];
    self.txtAdjust.text = str;
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
}

@end

File: swift/YuGiOhCard/Controller/CardAdjustViewController.swift
import UIKit

class CardAdjustViewController: UIViewController {

    @IBOutlet var txtAdjust: UITextView?
    var card: CardItem?
    
    var inited = false
    
    override func viewWillLayoutSubviews() {
        super.viewWillLayoutSubviews()
        if (!inited) {
            inited = true
            for temp in self.view.subviews {
                if (temp is UITextView) {
                    UIUtils.scaleComponent(view: temp)
                }
            }
        }
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        inited = false
        self.card = (self.tabBarController! as! CardViewController).card
        let str = "\(self.card!.adjust)\n"
        self.txtAdjust!.text = str
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }


}
